openmc/surface.py

```diff
@@ -139,65 +152,84 @@ def create_xml_subelement(self):
 
         element.set("type", self._type)
         element.set("boundary", self._boundary_type)
-
-        coeffs = ''
-
-        for coeff in self._coeff_keys:
-            coeffs += '{0} '.format(self._coeffs[coeff])
-
-        element.set("coeffs", coeffs.rstrip(' '))
+        element.set("coeffs", ' '.join(map(str, self._coeffs)))
```

Comment:
This is actually incorrect - this puts the dict keys into the XML element rather than its values. An appropriate fix might be:

```python
element.set("coeffs", ' '.join(map(str, self_coeffs.values()))
```

You can show this bug by trying to run any of the example python input files in "examples/python".